pkg/docker/docker.go

```diff
@@ -544,8 +545,11 @@ func runContainerTar(opts RunContainerOptions, imageMetadata *docker.Image) (cmd
 	return cmd, tarDestination
 }
 
-// this funtion simply abstracts out the running of the newly produced image when the --run=true option is provided
-func runContainerDockerRun(container *docker.Container, d *stiDocker, image string) {
+// dumpContainerInfoAndWait dumps information about a running container (port/ip/etc) and then
+// waits for the container to exit (which it may never do).  Used when --run=true is passed
```

Comment:
yeah i think you are correct, though i think that's still the same behavior it had before (previously it would block forever waiting for sigint, regardless of whether the container exited.  now it will block forever waiting for sigint/quit/term/hup, regardless of whether the container exited.

I think I should just update this method to do nothing but dump the container info, and then follow the normal flow into the "WaitContainer" logic.  Now that we have signal handling in place, that should be fine.